Bottom: ae995fc094110cf520a8dc855430e013e0b801e2
Top:    fa81ce17d57d144365ea60fcea5b02339c556a7a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 15:05:42 +0100

Refresh of hid-gtk-use-gtk_widget_get_win

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index b5fa405..85fd1d6 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -1331,7 +1331,7 @@ ghid_pcb_to_event_coords (Coord pcb_x, Coord pcb_y, int *event_x, int *event_y)
 static void
 draw_lead_user (render_priv *priv)
 {
-  GdkWindow *window = gtk_widget_get_window (widget);
+  GdkWindow *window = gtk_widget_get_window (gport->drawing_area);
   int i;
   Coord radius = priv->lead_user_radius;
   Coord width = MM_TO_COORD (LEAD_USER_WIDTH);
